util.go

```diff
@@ -96,13 +96,13 @@ func pushPullScale(interval time.Duration, n int) time.Duration {
 	return time.Duration(multiplier) * interval
 }
 
-// moveDeadNodes moves nodes that are dead and beyond the gossip to the dead interval
+// moveDeadNodes moves nodes that are dead and left and beyond the gossip to the dead interval
```

Comment:
I'm not sure I follow this godoc comment, is there an extra word or a missing word here?

Does this sound right ?

```suggestion
// moveDeadNodes moves dead and left nodes that that have not changed during the gossipToTheDeadTime duration
```

Reply:
As far as a test goes, maybe something in state_test:TestMemberList_ResetNodes?